scripts/benchmarks/benchstat.sh

```
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source "$(dirname "${BASH_SOURCE[0]}")/env.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./env.sh was not specified as input (see shellcheck -x).

(SC1091)